api/v1beta1/hetznerbaremetalmachine_types.go

```diff
@@ -265,6 +265,11 @@ type HetznerBareMetalMachineStatus struct {
 	// +optional
 	Ready bool `json:"ready"`
 
+	// Phase represents the current phase of HetznerBareMetalMachineStatus actuation.
+	// E.g. Pending, Running, Terminating, Failed etc.
```

Comment:
Terminating and failed is something we don't even use.. Maybe adjust this comment